addons/mail/static/src/models/user_setting/user_setting.js

```diff
@@ -256,6 +259,17 @@ function factory(dependencies) {
             }, 2000, 'globalSettings');
         }
 
+        /**
+         * @private
+         * @param {Event} ev
+         */
+        async _onStorage(ev) {
+            if (ev.key === 'mail_user_setting_voice_threshold') {
+                this.update({ voiceActivationThreshold: ev.newValue });
+                await this.messaging.rtc.updateVoiceActivation();
```

Comment:
Out-of-scope of PR, just wanted to point out that master code should eventually be changed so there's no need to manually call an `updateX` function like `Rtc/updateVoiceActivation`.

Ideally the update on the field should update everything else thanks to dependencies among records and fields.